pkg/controllermanager/controller/shoot/shoot_hibernation_types.go

```diff
@@ -123,9 +126,16 @@ func (h *hibernationJob) Run() {
 		return
 	}
 	h.logger.Debugf("Successfully set hibernation.enabled to %t", h.enabled)
+	if h.enabled {
+		msg := "Hibernating cluster"
+		h.recorder.Eventf(h.target, corev1.EventTypeNormal, gardencorev1beta1.ShootEventHibernationEnabled, "%s", msg)
+	} else {
+		msg := "Waking up cluster from hibernation"
```

Comment:
```suggestion
		msg := "Waking up cluster due to schedule"
```

We should make clear that these events do not capture the hibernation/wake-ups triggered by the user itself but only those triggered automatically due to a configured hibernation schedule.